tests/PHPStan/Analyser/looseConstComparisonPhp7.neon

```diff
@@ -0,0 +1,2 @@
+parameters:
+	phpVersion: 70400 # PHP 7.4
```

Comment:
when running the test-cases in isolation, outside of the phpstan repo they behave as expected. 

I somehow have the feeling that using `phpVersion` within the test-suite is kind of a unsupported feature in the current test infrastructure

Reply:
You need to change `new PhpVersion(PHP_VERSION_ID)` in PHPStanTestCase

Reply:
thank you.